app/Http/Controllers/ThirdParty/CallPowerController.php

```diff
@@ -35,7 +35,21 @@ public function store(Request $request)
             'number_dialed_into' => 'required',
         ]);
 
+        Log::debug('sending job to create post with details: ' . json_encode([
+                'mobile' => $request['mobile'],
+                'callpower_campaign_id' => $request['callpower_campaign_id'],
+                'status' => $request['status'],
+            ])
+        );
+
         // Send to queued job.
         CreateCallPowerPostInRogue::dispatch($parameters);
+
+        Log::debug('sent job to create post with details: ' . json_encode([
```

Comment:
This will only tell us how long it took for the job to be inserted into the queue (since the jobs are processed later by the queue dyno, not as part of the web request). From my understanding, we had problems processing jobs already in the queue, so this wouldn't tell us much.

From looking at the job, it seems like we actually already have good instrumentation that we can use for debugging! I'd recommend keeping one of these & then we should be good to go! 😸

Reply:
ok sounds good, thanks @DFurnes ! Yeah I wasn't sure how helpful these would be but I will keep the first one!